src/build/app/settings.rs

```diff
@@ -80,8 +79,8 @@ impl_settings! { AppSettings, AppFlags,
         => Flags::ARGS_NEGATE_SCS,
     AllowExternalSubcommands("allowexternalsubcommands")
         => Flags::ALLOW_UNK_SC,
-    AllowInvalidUtf8("allowinvalidutf8")
-        => Flags::UTF8_NONE,
+    AllowInvalidUtf8ExternalSubcommands("allowinvalidutf8externalsubcommands")
```

Comment:
I am thinking that we can probably find a better alternative for `AllowInvalidUtf8ExternalSubcommands`.

`AllowInvalidUtf8ForExternalSubcommand`?

Reply:
*shrug* Its already longer than / more verbose than I like to have in an API but I can see how different people can come to different conclusions, I'll change this.